### PR TITLE
Exec copy 2022-08-31

### PR DESCRIPTION
**Pull request by blimpa**
Includes drawdown of 25 million DAI for HVBank and Teleport launch on Optimism and Arbitrum

@prose11

### CODE REVIEW

File: governance/votes/Executive vote - August 31, 2022.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### HVBank (RWA-009) Drawdown
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), 25 million DAI will be drawn down in the RWA-009 (HVBank) vault, if this executive proposal passes.
+
+Further description of the schedule and mechanism being used for drawdowns from this vault can be seen in this [post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/8) from the Collateral Engineering Services Core Unit (CES-001).
+
+### Maker Teleport Launch on Optimism and Arbitrum
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmahjYA2#poll-detail), [Maker Teleport (formerly known as Maker Wormhole)](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) will be launched on Optimism and Arbitrum with a 1 basis point fee.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:44]: Debt ceiling? 1 million for each right?

File: governance/votes/Executive vote - August 31, 2022.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### HVBank (RWA-009) Drawdown
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), 25 million DAI will be drawn down in the RWA-009 (HVBank) vault, if this executive proposal passes.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:38]: Maybe add the following?

```suggestion
As per this successful governance [poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), 25 million DAI will be drawn down in the RWA-009 (HVBank) vault, if this executive proposal passes. This is the fourth and final planned drawdown for this vault.
```

File: governance/votes/Executive vote - August 31, 2022.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **change** will occur within the Maker Protocol:
+- 25 million DAI will be drawn down in the RWA-009 (HVBank) vault, as detailed below.
+
+If this executive proposal passes, the following **addition** will occur within the Maker Protocol:
+- Maker Teleport will be launched on Optimism and Arbitrum with a 1 basis point fee.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:22]: Debt ceiling?

File: governance/votes/Executive vote - August 31, 2022.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: Template - [Executive Vote] HVBank Drawdown, Maker Teleport Launch on Optimism and Arbitrum - August 31, 2022
+summary: Includes drawdown of 25 million DAI for HVBank and Teleport launch on Optimism and Arbitrum.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:3]: Gotta keep the branding 😜 

```suggestion
summary: Includes drawdown of 25 million DAI for HVBank and Maker Teleport launch on Optimism and Arbitrum.
```

File: governance/votes/Executive vote - August 31, 2022.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), 25 million DAI will be drawn down in the RWA-009 (HVBank) vault, if this executive proposal passes.
+
+Further description of the schedule and mechanism being used for drawdowns from this vault can be seen in this [post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/8) from the Collateral Engineering Services Core Unit (CES-001).
+
+### Maker Teleport Launch on Optimism and Arbitrum
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmahjYA2#poll-detail), [Maker Teleport (formerly known as Maker Wormhole)](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) will be launched on Optimism and Arbitrum with a 1 basis point fee.
+
+The Maker Teleport initiative will be bringing canonical DAI capability onto L2 domains.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:46]:
```suggestion
The Maker Teleport initiative will bring canonical DAI capability onto L2 domains.
```

File: governance/votes/Executive vote - August 31, 2022.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+For more details, please read the [Signal Request from which the passed governance poll spawned off](https://forum.makerdao.com/t/signal-request-launch-maker-teleport-with-1-basis-point-fee/15109) and the [Maker Teleport: Oracle Launch Configuration post](https://forum.makerdao.com/t/maker-teleport-oracle-launch-configuration/17471). Forum threads related to the Teleport initiative can be found [here](https://forum.makerdao.com/tag/teleport).
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - August 31, 2022.md:48]: Wording is a bit clunky IMO. Maybe change to:

```suggestion
For more details, please review the original [Signal Request](https://forum.makerdao.com/t/signal-request-launch-maker-teleport-with-1-basis-point-fee/15109) that led to the governance poll, and the [Maker Teleport: Oracle Launch Configuration post](https://forum.makerdao.com/t/maker-teleport-oracle-launch-configuration/17471). Forum threads related to the Teleport initiative can be found [here](https://forum.makerdao.com/tag/teleport).
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

**Review by prose11 — APPROVED** [review]
Nice work everyone :)

**Pull request merged**